Cython/Compiler/Parsing.py

```diff
@@ -2337,14 +2338,25 @@ def p_statement(s, ctx, first_statement = 0):
 
 def p_statement_list(s, ctx, first_statement = 0):
     # Parse a series of statements separated by newlines.
+    global stat
```

Comment:
Why is this added?

Reply:
because without it, stat raises an internal error of "reference before assignment" inside the try catch on every statement check inside the loop through statements

-> error raised: "Internal Error: local variable 'stat' referenced before assignment"

I thought that this error should not be a thing to be raised with the other ones...

Reply:
This isn't right. Please remove that.

I believe the proper fix is adding a 

`continue`

right after the `report_error`.

Reply:
p.s.: if the `continue` doesn't work we can discuss a bit more about it...

Reply:
Actually, it should be after the `report_error`, but after the `s.next()`.

Reply:
it worked fine ;)